### ISSUE TITLE
commands.Bot() is not work in 2.0.0b4

### ISSUE THREAD

**Issue by cheap122000** [Pycord-Development/pycord#1042]
### Summary

"Traditional Commands Example" on README.md is not worked.

### Reproduction Steps

```py
import discord
from discord.ext import commands

bot = commands.Bot(command_prefix=">")

@bot.command()
async def ping(ctx):
    await ctx.send("pong")

bot.run("token")
```

### Expected Results

When I type ">ping" on discord and it will response me "pong"

### Actual Results

Nothing happened.

### Intents

None

### System Information

- Python v3.9.7-final
- py-cord v2.0.0-beta
    - py-cord pkg_resources: v2.0.0b4
- aiohttp v3.7.4.post0
- system info: Darwin 21.1.0 Darwin Kernel Version 21.1.0: Wed Oct 13 17:33:23 PDT 2021; root:xnu-8019.41.5~1/RELEASE_X86_64

### Checklist

- [X] I have searched the open issues for duplicates.
- [X] I have shown the entire traceback, if possible.
- [X] I have removed my token from display, if visible.

### Additional Context

I finally solved it by downgrade py-cord 2.0.0b4 to 2.0.0b3.

**Comment by EmreTech** [comment]
Can you please share the errors you got in the terminal when attempting to run the code if there is any?

**Comment by EmreTech** [comment]
Actually, nevermind about that last comment. This is caused because the example has not been updated for message content intent. You must enable it in order to read the contents from messages.

**Comment by FaberSid** [comment]
I tried one commit at a time...
It worked on b031d749047b1cb596e4a1105b3aa7520d99045e, but not on 19226949b8230993644ae19d53f59140eb5f4c48.
I don't think I can write any more about it because I couldn't understand why this happens even after looking at the commit.
It may not be related, but I'll leave it as information.

**Comment by EmreTech** [comment]
You should probably look at the contents of the commit that doesn't work. It adds the message content intent. Enable it.

**Comment by EmreTech** [comment]
The current latest commit does add this disclaimer, that you have to enable the message content intent in order to use prefixed commands.

**Comment by krittick** [comment]
Closing as this is addressed per the above comments re: #1014
